package.json

```diff
@@ -20,8 +20,8 @@
   "main": "sparql.js",
   "bin": "bin/sparql-to-json",
   "scripts": {
-    "build": "jison lib/sparql.jison -p slr -o lib/SparqlParser.js",
-    "test": "mocha"
+    "build": "./node_modules/.bin/jison lib/sparql.jison -p slr -o lib/SparqlParser.js",
+    "test": "./node_modules/.bin/mocha"
```

Comment:
This shouldn't be necessary; npm should take care of that.
@joachimvh As a Windows user, can you comment?

Reply:
`npm run-script build` and `npm test` work for me from the command line without having to change anything. I do have `mocha` installed as global dependency (but not `jison` so that shouldn't make a difference)

Reply:
@AlexeyMz Any insight in why it wouldn't work on your system?